report/src/main/java/com/hartwig/actin/report/pdf/ReportWriter.java

```diff
@@ -27,6 +30,7 @@
 
 public class ReportWriter {
 
+    public static final float STANDARD_KEY_WIDTH = 210;
```

Comment:
This property does not really belong here. I think "Formats" may be a better location?